tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -144,6 +144,7 @@ public ICollection<TelemetryValue> GetConfigurationData()
                 new(ConfigTelemetryData.CodeHotspotsEnabled, value: _profiler?.ContextTracker.IsEnabled),
                 new(ConfigTelemetryData.StatsComputationEnabled, value: settings.StatsComputationEnabled),
                 new(ConfigTelemetryData.WcfObfuscationEnabled, value: settings.WcfObfuscationEnabled),
+                new(ConfigTelemetryData.DataStreamsMonitoringEnabled, value: settings.IsDataStreamsMonitoringEnabled),
```

Comment:
You need to update line 119 IIRC where we set the capacity of the list

Reply:
( As this code isn't triggered too much, I even wonder if we should remove passing the capacity as we will keep forgetting it. )

Reply:
🤦‍♂️ thanks, I'll update it. It's annoying, that's for sure, but I think we should keep it and be rigorous about updating 😄 As long as a reviewer spots it, it's worth it 😉